src/templates/common/.github/workflows/Deploy.yml

```diff
@@ -24,18 +24,19 @@ jobs:
       uses: julia-actions/setup-julia@v1
       with:
         version: 1.5
+    - name: Install dependencies listed in Project.toml and build package
+      uses: julia-actions/julia-buildpkg@master
     # NOTE
-    #   The steps below ensure that NodeJS and Franklin are loaded then it
-    #   installs highlight.js which is needed for the prerendering step
+    #   Using --project below ensures that NodeJS and Franklin are loaded then 
+    #   it installs highlight.js which is needed for the prerendering step
```

Comment:
This comment (re NodeJS and Franklin) is now incorrect right? what it means is that it'll ensure that whatever you have in your Project.toml is installed, correct?

Reply:
The word "activated" might be better indeed. `julia --project` is identical to `julia -e 'using Pkg; Pkg.activate(".")` as far as I know.

Reply:
ok cool, I'll merge this and do light edits to reflect this